### PR TITLE
check starting vertex in WanderingCommander

### PR DESCRIPTION
**Pull request by yingjieMiao**
@leogrim @stkem 

this is a cheap solution before we go to job queue...

### CODE REVIEW

File: kifi-backend/modules/graph/app/com/keepit/graph/wander/WanderingCommander.scala [hunk 1]
```diff
@@ -50,7 +50,11 @@ class WanderingCommander @Inject() (graph: GraphManager, clock: Clock) extends L
         val wanderer = reader.getNewVertexReader()
         val scout = reader.getNewVertexReader()
         val scoutingWanderer = new ScoutingWanderer(wanderer, scout)
-        scoutingWanderer.wander(wanderlust.steps, teleporter, resolver, journal)
+        if (wanderer.hasVertex(startingVertexId)) {
+          scoutingWanderer.wander(wanderlust.steps, teleporter, resolver, journal)
+        } else {
+          log.error(s"trying to start a random walk from non-existing vertex: ${startingVertexKind}: ${wanderlust.startingVertexDataId}. Return empty journal now.")
```
Comment (stkem) [inline kifi-backend/modules/graph/app/com/keepit/graph/wander/WanderingCommander.scala:56]: Could returning an empty result be problematic with the PYMK caching?

Comment (yingjieMiao) [inline kifi-backend/modules/graph/app/com/keepit/graph/wander/WanderingCommander.scala:56]: hmmm, I assume PYMK uses `SocialWanderingCommander`. This fix should only affect uriWandering problem. But this is a good point.

Comment (stkem) [inline kifi-backend/modules/graph/app/com/keepit/graph/wander/WanderingCommander.scala:56]: oh, I think you are right. Just thought I'd bring it up. Can you confirm @leogrim

Comment (leogrim) [inline kifi-backend/modules/graph/app/com/keepit/graph/wander/WanderingCommander.scala:56]: Yep, @yingjieMiao is right :)

**Comment by yingjieMiao** [comment]
thanks for the feedback. merge this, hope to reduce some production noise.

**Pull request merged**